packages/dd-trace/src/platform/node/metrics.js

```diff
@@ -222,49 +272,52 @@ function captureNativeMetrics () {
   const systemPercent = 100 * stats.cpu.system / elapsedUs
   const totalPercent = userPercent + systemPercent
 
-  client.gauge('cpu.system', systemPercent.toFixed(2))
-  client.gauge('cpu.user', userPercent.toFixed(2))
-  client.gauge('cpu.total', totalPercent.toFixed(2))
+  client.gauge('runtime.node.cpu.system', systemPercent.toFixed(2))
+  client.gauge('runtime.node.cpu.user', userPercent.toFixed(2))
+  client.gauge('runtime.node.cpu.total', totalPercent.toFixed(2))
 
-  histogram('event_loop.delay', stats.eventLoop)
+  histogram('runtime.node.event_loop.delay', stats.eventLoop)
 
   Object.keys(stats.gc).forEach(type => {
     if (type === 'all') {
-      histogram('gc.pause', stats.gc[type])
+      histogram('runtime.node.gc.pause', stats.gc[type])
     } else {
-      histogram('gc.pause.by.type', stats.gc[type], [`gc_type:${type}`])
+      histogram('runtime.node.gc.pause.by.type', stats.gc[type], [`gc_type:${type}`])
     }
   })
 
-  client.gauge('spans.finished', stats.spans.total.finished)
-  client.gauge('spans.unfinished', stats.spans.total.unfinished)
+  client.gauge('runtime.node.spans.finished', stats.spans.total.finished)
+  client.gauge('runtime.node.spans.unfinished', stats.spans.total.unfinished)
 
   for (let i = 0, l = spaces.length; i < l; i++) {
     const tags = [`heap_space:${spaces[i].space_name}`]
 
-    client.gauge('heap.size.by.space', spaces[i].space_size, tags)
-    client.gauge('heap.used_size.by.space', spaces[i].space_used_size, tags)
-    client.gauge('heap.available_size.by.space', spaces[i].space_available_size, tags)
-    client.gauge('heap.physical_size.by.space', spaces[i].physical_space_size, tags)
+    client.gauge('runtime.node.heap.size.by.space', spaces[i].space_size, tags)
+    client.gauge('runtime.node.heap.used_size.by.space', spaces[i].space_used_size, tags)
+    client.gauge('runtime.node.heap.available_size.by.space', spaces[i].space_available_size, tags)
+    client.gauge('runtime.node.heap.physical_size.by.space', spaces[i].physical_space_size, tags)
   }
 
   if (stats.spans.operations) {
     const operations = stats.spans.operations
 
     Object.keys(operations.finished).forEach(name => {
-      client.gauge('spans.finished.by.name', operations.finished[name], [`span_name:${name}`])
+      client.gauge('runtime.node.spans.finished.by.name', operations.finished[name], [`span_name:${name}`])
     })
 
     Object.keys(operations.unfinished).forEach(name => {
-      client.gauge('spans.unfinished.by.name', operations.unfinished[name], [`span_name:${name}`])
+      client.gauge('runtime.node.spans.unfinished.by.name', operations.unfinished[name], [`span_name:${name}`])
     })
   }
 }
 
 function histogram (name, stats, tags) {
+  tags = [].concat(tags)
+
   client.gauge(`${name}.min`, stats.min, tags)
   client.gauge(`${name}.max`, stats.max, tags)
   client.increment(`${name}.sum`, stats.sum, tags)
```

Comment:
Still curious what happens with this if the agent is configured to enable `.sum`

Reply:
Not a problem since they are not reported as histograms. If we switch to compute in the agent however, it will be a breaking change for existing dashboards.

Reply:
oh duh... thanks.